javascript/react/good-to-know/immutability-helpers-in-react.md

```diff
@@ -115,4 +141,9 @@ var newObj = ???(myData, {
 * `{`
 * `}`
 
+---
+
```

Comment:
There shouldn't be any new line between `---` and `##`

Reply:
👍 I'm in the process of fixing all the rule issues